TDAT: public throwable assert has no cause ( ) {  <NL> is not null ( ) ;  <NL> throwable actual cause = actual . get cause ( ) ;  <NL> if ( actual cause ! = null )  <NL> fail ( concat ( " expected exception without cause , but cause was : " , in brackets ( actual cause . get class ( ) ) ) ) ;  <NL> return this ;  <NL>  }  COM: <s> verifies that the actual code throwable code does not have a cause </s>